packages/libbinaryen/libbinaryen.106.0.0/opam

```diff
@@ -0,0 +1,28 @@
+opam-version: "2.0"
+synopsis: "Libbinaryen packaged for OCaml"
+maintainer: "blaine@grain-lang.org"
+authors: "Blaine Bublitz"
+license: "Apache-2.0"
+homepage: "https://github.com/grain-lang/libbinaryen"
+bug-reports: "https://github.com/grain-lang/libbinaryen/issues"
+depends: [
+  "conf-cmake" {build}
+  "dune" {>= "2.9.1" & < "3.0.0"}
+  "dune-configurator" {>= "2.9.1" & < "3.0.0"}
+  "js_of_ocaml-compiler" {with-test & >= "3.10.0" & < "4.0.0"}
+  "ocaml" {>= "4.12"}
+]
+build: [
+  ["dune" "subst"] {dev}
+  ["dune" "build" "-p" name "-j" jobs]
+]
+depexts: ["gcc-g++"] {os-distribution = "cygwinports"}
+dev-repo: "git+https://github.com/grain-lang/libbinaryen.git"
```

Comment:
```suggestion
available: arch != "arm32" & arch != "x86_32"
dev-repo: "git+https://github.com/grain-lang/libbinaryen.git"
```

Reply:
Thanks @mseri - This is definitely an upstream Binaryen issue. I'm trying to catch up to their releases (106 and 107), so let's add this here. I won't add "available" to my 107 release so we can see if the issue/bug is still present and if so, I'll open an issue upstream.